mailscanner/languages/de.php

```diff
@@ -515,12 +515,12 @@
     'topsendersdomvol14' => 'Top Absender-Domänen nach Volumen',
     'toprecipdomqt14' => 'Top Empfänger-Domänen nach Anzahl',
     'toprecipdomvol14' => 'Top Empfänger-Domänen nach Volumen',
-    'assassinscoredist14' => 'SpamAssassin Score Distribution',
-    'assassinrulhit14' => 'SpamAssassin Rule Hits',
-    'auditlog14' => 'Audit Log',
-    'mrtgreport14' => 'MRTG Style Report',
-    'mcpscoredist14' => 'MCP Score Distribution',
-    'mcprulehit14' => 'MCP Rule Hit',
+    'assassinscoredist14' => 'Verteilung der SpamAssassin-Bewertungen',
+    'assassinrulhit14' => 'Angewandte SpamAssassin-Regeln',
+    'auditlog14' => 'Kontrollbericht',
+    //'mrtgreport14' => 'MRTG Style Report', unused
```

Comment:
It would be nice to fix it, it should generate a graph of last 24 hours showing a sum of all email, spam and virii grouped by hour (like 24 columns from 30/05/17 18:00 to 31/05/17 18:00)